rotkehlchen/tests/unit/decoders/accounting_settings/test_default_settings.py

```diff
@@ -0,0 +1,271 @@
+from unittest.mock import patch
+
+import pytest
+
+from rotkehlchen.accounting.cost_basis.base import (
+    AssetAcquisitionEvent,
+    CostBasisInfo,
+    MatchedAcquisition,
+)
+from rotkehlchen.accounting.mixins.event import AccountingEventType
+from rotkehlchen.accounting.pnl import PNL
+from rotkehlchen.accounting.pot import AccountingPot
+from rotkehlchen.accounting.structures.balance import Balance
+from rotkehlchen.accounting.structures.base import HistoryBaseEntry
+from rotkehlchen.accounting.structures.processed_event import ProcessedAccountingEvent
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.chain.evm.decoding.constants import CPT_GAS
+from rotkehlchen.chain.evm.types import string_to_evm_address
+from rotkehlchen.constants import ZERO
+from rotkehlchen.constants.assets import A_ETH
+from rotkehlchen.constants.misc import ONE
+from rotkehlchen.fval import FVal
+from rotkehlchen.tests.utils.history import prices
+from rotkehlchen.types import Location, Timestamp, TimestampMS, deserialize_evm_tx_hash
+
+
+@pytest.fixture(name='pot')
+def fixture_pot(accountant):
+    pot: 'AccountingPot' = accountant.pots[0]
+    with pot.database.conn.read_ctx() as cursor:
+        settings = pot.database.get_settings(cursor)
+    pot.reset(
+        settings=settings,
+        start_ts=Timestamp(0),
+        end_ts=Timestamp(100),
+        report_id=1,
+    )
+    return pot
+
+
+def gain_one_ether(transactions, evmhash):
+    eth_gain_event = HistoryBaseEntry(
+        event_identifier=evmhash,
+        sequence_index=0,
+        timestamp=TimestampMS(1624395186000),
+        location=Location.BLOCKCHAIN,
+        location_label=string_to_evm_address('0x4bBa290826C253BD854121346c370a9886d1bC26'),
+        asset=A_ETH,
+        balance=Balance(amount=ONE),
+        notes='Received 1 ETH',
+        event_type=HistoryEventType.RECEIVE,
+        event_subtype=HistoryEventSubType.NONE,
+    )
+    consumed_num = transactions.process(event=eth_gain_event, events_iterator=None)
+    assert consumed_num == 1
+
+
+def test_accounting_no_settings(pot):
+    """Test that if there are no settings provided, the event is not taken into account"""
+    tx_hex = '0x282f632c7a2385d026068122dc415562a1c4e827fd54673671af1d40445d6b58'
+    evmhash = deserialize_evm_tx_hash(tx_hex)
+    event = HistoryBaseEntry(
+        event_identifier=evmhash,
+        sequence_index=0,
+        timestamp=TimestampMS(0),
+        location=Location.BLOCKCHAIN,
+        event_type=HistoryEventType.INFORMATIONAL,
+        event_subtype=HistoryEventSubType.NONE,
+        asset=A_ETH,
+        balance=Balance(),
+    )
+    was_skipped = False
+
+    def log_mock(text):
+        assert 'Skipping...' in text
+        nonlocal was_skipped
+        was_skipped = True
+
+    log_patch = patch('rotkehlchen.accounting.transactions.log.debug', side_effect=log_mock)
+    with log_patch:
```

Comment:
can it go in one line? if you don't need to reuse it I believe is fine to just do

```
with patch(...
```

Reply:
Alright. Wanted to be more consistent with the usages of patches in other tests, but I guess can also go in one line